lib/spack/spack/cmd/deconcretize.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import sys
+
+import llnl.util.tty as tty
+
+import spack.cmd
+import spack.cmd.common.arguments as arguments
+import spack.cmd.common.confirmation as confirmation
+import spack.spec
+
+description = "remove specs from the concretized lockfile of an environment"
+section = "environments"
+level = "long"
+
+# Arguments for display_specs when we find ambiguity
+display_args = {"long": True, "show_flags": False, "variants": False, "indent": 4}
+
+
+def setup_parser(subparser):
+    subparser.add_argument(
+        "--root",
+        action="store_true",
+        help="deconcretize only environment roots. additional roots that depend on matching specs"
+        " will be ignored",
```

Comment:
This wraps around on an 80-col terminal 

```suggestion
        help="deconcretize only specific environment roots",
```